timestamp-impl/src/main/java/com/palantir/timestamp/AvailableTimestamps.java

```diff
@@ -87,7 +90,7 @@ private synchronized TimestampRange handOutTimestamp(long targetTimestamp) {
     private void allocateEnoughTimestampsToHandOut(long timestamp) {
         log.trace("Increasing limit to at least {}.", timestamp);
         upperLimit.increaseToAtLeast(timestamp);
-        log.trace("Increasing done. Limit is now {}.", timestamp);
+        log.trace("Increased to at least {}. Limit is now {}.", timestamp, getUpperLimit());
```

Comment:
might want to consider either wrapping this one in `if (log.isTraceEnabled())` or just lazily `log.trace("Increased to at least {}. Limit is now {}.", timestamp, upperLimit);`